### PR TITLE
Fix small details

### PR DESCRIPTION
**Pull request by joseguti84**
* Refactor to this PR: https://github.com/CommonJobs/CommonJobs/pull/424
* Add validation in calificator manager

### CODE REVIEW

**Comment by sofipacifico** [comment]
:+1:

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js [hunk 1]
```diff
@@ -26,18 +26,20 @@
...
-                    return e.userName == userName;
-                }, this);
-                if (calificator) {
-                    calificator.action(0);
-                } else {
-                    var calificator = new Calificator();
-                    calificator.add(userName);
-                    self.calificators.push(calificator);
+            if (userName){
+                if (userName != self.evaluation.userName) {
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js]: What do you think about:

```
if (userName  && userName != self.evaluation.userName) {
```

**Comment by Gkolocsar** [comment]
Looks good to my untrained front-end eyes :)

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js (hunk 1 above)
Comment (joseguti84) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js]: I agree!

**Pull request merged**